### PR TITLE
site: Use a different background color for users logo

### PR DESCRIPTION
**Pull request by Dol0res**
issue-number：#162

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# 默认忽略的文件
```
Comment (chaokunyang) [inline .idea/.gitignore:1]: Could we use english for comments

**Comment by chaokunyang** [comment]
Could we delete .idea from commits?

File: .idea/.gitignore (hunk 1 above)
Comment (Dol0res) [reply .idea/.gitignore:1]: Sure

**Comment by Dol0res** [comment]
> Could we delete .idea from commits?

Sure, and  sorry for this inconvenience my oversight have caused you.

**Review by chaokunyang — APPROVED** [review]
LGTM, thanks

**Pull request merged**